e2e/config.js

```diff
@@ -6,9 +6,11 @@
  */
 
 module.exports = {
+  // TODO: Change back to https://scaffold-pwa-e2e-tests-pwa-kit.mobify-storefront.com
+  // when v3 is officially released
   RETAIL_APP_HOME:
     process.env.RETAIL_APP_HOME ||
-    "https://scaffold-pwa-e2e-tests-pwa-kit.mobify-storefront.com",
+    "https://scaffold-pwa-test-env-2.mobify-storefront.com",
```

Comment:
Since these new tests are not going to run nightly, I think we don't need to update this line. 

Maybe we can replace it with a comment that tells others about that `env-2` environment.

Reply:
If I don't change it, CI won't pass 🤔

Reply:
Created a ticket to make sure we don't forget that